en/config_mc/pid_tuning_guide_multicopter_basic.md

```diff
@@ -0,0 +1,93 @@
+- You should have done an [ESC calibration](../advanced_config/esc_calibration.md).
+- If using PWM output: [PWM_MIN](../advanced_config/parameter_reference.md#PWM_MIN) is set correctly.
+  It needs to be set low, but such that the **motors never stop** when the vehicle is armed.
+
+  This can be tested in [Acro mode](../flight_modes/acro_mc.md) or in [Manual/Stabilized mode](../flight_modes/manual_stabilized_mc.md):
+  - Remove propellers
+  - Arm the vehicle and lower the throttle to the minimum
+  - Tilt the vehicle to all directions, about 60 degrees
+  - Check that no motors turn off
+- Use a high-rate telemetry link such as WiFi if at all possible (a typical low-range telemetry radio is not fast enough for real-time feedback and plots).
+  This is particularly important for the rate controller.
+- Disable [MC_AIRMODE](../advanced_config/parameter_reference.md#MC_AIRMODE) before tuning a vehicle.
+
+## Tuning Procedure
+
+The tuning procedure is:
+
+1. Open _QGroundControl_ **Vehicle Setup > PID Tuning**
+   ![QGC Rate Controller Tuning UI](../../assets/mc_pid_tuning/qgc_mc_pid_tuning_rate_controller.png)
+1. Select the **Rate Controller** tab.
+1. Confirm that the airmode selector is set to **Disabled**
+1. Set the *Thrust curve* value to: ????
+1. Set the *Select Tuning* radio button to: **Roll**.
+1. For rate controller tuning, switch to *Acro mode*, *Stabilized mode* or *Altitude mode*.
+1. Select the **Start** button.
+1. Rapidly move the *roll stick* full range and observe the step response on the plots.
+1. Modify the three PID values using the sliders (for roll rate-tuning these affect `MC_ROLLRATE_K`, `MC_ROLLRATE_I`, `MC_ROLLRATE_D`) and observe the step response again.
+   :::note
+   The goal is for the _Response_ curve to match the _Setpoint_ curve as closely as possible (i.e. a fast response without overshoots).
+   :::
+   The PID values can be adjusted as follows:
+   - P (proportional) or K gain:
+     - increase this for more responsiveness
+     - reduce if the response is overshooting and/or oscillating (up to a certain point increasing the D gain also helps).
+   - D (derivative) gain:
+     - this can be increased to dampen overshoots and oscillations
+     - increase this only as much as needed, as it amplifies noise (and can lead to hot motors)
+   - I (integral) gain:
+     - used to reduce steady-state error
+     - if too low, the response might never reach the setpoint (e.g. in wind)
+     - if too high, slow oscillations can occur
+1. Repeat the tuning process above for the pitch and yaw:
+   - Use *Select Tuning* radio button to select the axis to tune
+   - Move the appropriate sticks (i.e. pitch stick for pitch, yaw stick for yaw).
+   - For pitch tuning, start with the same values as for roll.
+1. Repeat the tuning process for the attitude controller on all the axes.
+1. Repeat the tuning process for the velocity and positions controllers (on all the axes).
```

Comment:
Now we can move the tip from above (`Make sure to switch to the simple input mode, for direct control and possibility to generate step inputs`) down here

Reply:
Done. But how do you switch to simple input mode?

Reply:
It's in the UI:
![image](https://user-images.githubusercontent.com/281593/110756399-6e9e7000-824a-11eb-80f0-2575a241d562.png)

Reply:
Thanks. Was not obvious to me. Added in fixes.